artifactory.py

```diff
@@ -666,15 +667,18 @@ def rest_post(
         """
         Perform a POST request to url with requests.session
         """
-        res = session.post(
+        response = session.post(
             url,
             params=params,
             headers=headers,
             verify=verify,
             cert=cert,
             timeout=timeout,
         )
-        return res.text, res.status_code
+        if response.status_code not in (200, 201):
+            raise RuntimeError(response.text)
```

Comment:
`RuntimeError` is it right to raise this error? We have `ArtifactoryException` class for an error

Reply:
It is something that I moved from existing code, I agree, let's change it own exception

Reply:
>It is something that I moved from existing code

Oh, I'm sorry, I saw it too late :) No, actually we can save it as is! Or change - if you don't mind, it's up to you

Reply:
`raise_for_status()` should do a job for us actually. It will raise HTTP exception with appropriate message for unsuccessful status codes